modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-disk-sharing-lun_{context}"]
+= Configuring disk sharing by using LUN
+
+You can configure a LUN-backed block mode virtual machine disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling this option allows you to use advanced SCSI commands, such as those required for a Windows Failover Clustering implementation, against the underlying storage. 
+
+A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough on the disk. 
+
+You reserve a LUN through the SCSI Persistent Reserve options to protect data on the VM from outside access. In order to issue privileged, SCSI device-specific input and output controls (IOCTLs), the VM requires activation of the `PersistentReservation` option on the LUN disk. You then enable the reservation by configuring the `featureGates` option. 
+
+.Prerequisites
+
+* `ReadWriteMany` (RWX) volume access mode is required if the VMs that are sharing disks are running on different nodes. If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
+
+
+.Procedure
+
+. Edit or create the `VirtualMachine` manifest for your VM to set the required values, as shown in the following example: 
+
++
+[source,yaml]
+====
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+    name: testvmi-lun
+spec:
+    template:
+        spec:
+            domain:
+    resources:
+      requests:
+        memory: 64M
+    devices:
+          disks:
+          - disk:
+               bus: virtio
+              name: rootdisk
+            disk: {} 
+            shareable: true <1>
+          - disk:
+               bus: virtio
+              name: cloudinitdisk
+                  lun: {} <2>
+                  bus: scsi
+                  reservation: true <3>
+# ...
+configuration:
+    developerConfiguration:
+        featureGates:
+            PersistentReservation <4>
```

Comment:
as the cluster admin you are not supposed to directly amend the Kubevirt CR that is managed by HCO:
you are only supposed to configure the HCO CR to expose this FG. Something like:

```
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
spec:
...
  featureGates:
    persistentReservation: true
```
Then please notice that we are talking about two different personas here:
the cluster admin is supposed to amend the cluster wide configuration on the HCO level, then the VM owner can continue on the VirtualMachine in his namespace.